install-android-sdk.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+SDKDIR=$PWD/.android-sdk
+export ANDROID_SDK_ROOT=${SDKDIR}
+export ANDROID_HOME=${SDKDIR}
+export ANDROID_AVD_HOME=${SDKDIR}/avd
+
+mkdir ${SDKDIR}
+mkdir ${SDKDIR}/cmdline-tools
+
+echo Downloading Android SDK...
+cd ${SDKDIR}/cmdline-tools
+COMMAND_LINE_TOOLS_ZIP=${SDKDIR}/commandlinetools.zip
+# https://developer.android.com/studio#command-tools
+TYPE=linux
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  TYPE=mac
+fi
+curl -q https://dl.google.com/android/repository/commandlinetools-${TYPE}-8512546_latest.zip -o ${COMMAND_LINE_TOOLS_ZIP}
+unzip ${COMMAND_LINE_TOOLS_ZIP}
+rm ${COMMAND_LINE_TOOLS_ZIP}
+mv cmdline-tools latest
+ln -s ${SDKDIR}/cmdline-tools/latest ${SDKDIR}/tools
+
+echo Installing emulator...
+yes | ${ANDROID_HOME}/tools/bin/sdkmanager --install platform-tools emulator
+
+echo Installing platform SDK...
+yes | ${ANDROID_HOME}/tools/bin/sdkmanager --install "platforms;android-33"
+
+echo Starting ADB...
```

Comment:
I think we also don't need to start the ADB since we're not going to connect to android devices